### PR TITLE
add json2html@1.0.0 w/ git auto-update

### PR DESCRIPTION
**Pull request by sufuf3**
Pull request for issue: #10292
Related issue(s): # #
@cdnjs/intern3 please help me review this PR, thank you.
Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/moappi/json2html
 * Official website (optional, not the repository): http://www.json2html.com
 * NPM package url (optional): https://www.npmjs.com/package/node-json2html
 * License and its reference: ISC
 * GitHub / Bitbucket popularity (required):
   - Count of watchers: 33
   - Count of stars: 304
   - Count of forks: 68
 * NPM download stats (optional):
   - Downloads in the last day: 31
   - Downloads in the last week: 185
   - Downloads in the last month: 862

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [x] More than 500 downloads stats per month on npm registry
 * [ ] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

**Comment by sufuf3** [comment]
I saw ISC license from https://www.npmjs.com/package/node-json2html 
and the description between ISC and MIT are similar.

**Comment by hare1039** [comment]
LGTM

**Comment by sufuf3** [comment]
@kennynaoh please help me review this PR, thank you.

**Comment by kennynaoh** [comment]
@sufuf3 I think it shows MIT license on website http://www.json2html.com/. You can confirm it here https://spdx.org/licenses/MIT.html#licenseText, but it shows ISC on npm package. I think we should confirm with author.

**Comment by sufuf3** [comment]
@kennynaoh I modified it.
It's MIT license, because the author pull the request for issue with the MIT license.
Please help me review this PR, thank you.

**Comment by kennynaoh** [comment]
Sorry, I can't understand. Any references? Thanks!

**Comment by sufuf3** [comment]
#10292 this is the author open the issue which show the MIT License.

**Comment by kennynaoh** [comment]
@PeterDaveHello Should we discuss with author about the license? It shows `MIT` on website http://www.json2html.com/, but `ISC` on npm https://www.npmjs.com/package/node-json2html

**Comment by PeterDaveHello** [comment]
Why do you use the `node` wrapper json2html?

**Comment by sufuf3** [comment]
Because there are no tags from git repo.

**Comment by PeterDaveHello** [comment]
Then you should ask if the author can add tag for it, not to use something looks not suitable, it's very clear that it said node-json2html is `for server side functionality`.

**Comment by sufuf3** [comment]
@PeterDaveHello @kennynaoh I modified it.
Please help me review this PR again, thank you.

File: ajax/libs/json2html/package.json [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    "templating",
+    "html",
+    "web",
+    "templates",
+    "transforms",
+    "json"
+  ],
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/moappi/node-json2html.git",
```
Comment (pvnr0082t) [inline ajax/libs/json2html/package.json]: Why do we fetch files from `node-json2html`? Should it be `json2html`?

File: ajax/libs/json2html/package.json [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    "name": "Chad Brown",
+    "email": "info@json2html.com",
+    "url": "http://json2html.com"
+  },
+  "description": "node-json2html - HTML Templating using JSON2HTML",
+  "version": "1.0.0",
+  "homepage": "http://json2html.com",
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/moappi/node-json2html.git"
```
Comment (pvnr0082t) [inline ajax/libs/json2html/package.json]: `node-json2html`? or `json2html`?

File: ajax/libs/json2html/package.json [hunk 3]
```diff
@@ -0,0 +1,38 @@
+{
+  "name": "json2html",
+  "filename": "json2html.min.js",
+  "author": {
+    "name": "Chad Brown",
+    "email": "info@json2html.com",
+    "url": "http://json2html.com"
+  },
+  "description": "node-json2html - HTML Templating using JSON2HTML",
```
Comment (pvnr0082t) [inline ajax/libs/json2html/package.json]: It seems like the description from `node-json2html`...

**Comment by sufuf3** [comment]
@pvnr0082t  I modified it.
Please help me review this PR again, thank you.

**Comment by sufuf3** [comment]
@pvnr0082t I delete origin `json2html.js` and add the files form git repo.
Please help me review this PR again, thank you.

**Comment by pvnr0082t** [comment]
@sufuf3 Just make a confirmation, did you do the minify again after you updated the `json2html.js`?

**Comment by sufuf3** [comment]
@pvnr0082t Yes.

**Comment by pvnr0082t** [comment]
ping @kennynaoh

**Comment by PeterDaveHello** [comment]
@sufuf3 please find out the reason why you added the wrong files and think about how to prevent it appears again, it's a dangerous mistake.

**Comment by PeterDaveHello** [comment]
@moappi would you please confirm this is what you expected? (Adding the json2html core? Or you want jquery wrapper?) Thanks.

**Comment by sufuf3** [comment]
@PeterDaveHello @moappi want add both of them.
ref: https://github.com/moappi/json2html/issues/24#issuecomment-281721720

**Comment by PeterDaveHello** [comment]
@sufuf3 Great!

**Comment by PeterDaveHello** [comment]
@sufuf3 please rebase with the latest master branch so I can merge it.

**Comment by sufuf3** [comment]
@PeterDaveHello I rebase with the latest master branch. Thank you!

**Comment by PeterDaveHello** [comment]
@sufuf3 don't forget to update your outdated fisrt comment

**Pull request merged**